### PR TITLE
Html task

### PR DESCRIPTION
**Pull request by Rybchynskyi**
### HTML & CSS Practice
[Demo](https://rybchynskyi.github.io/HTML-CSS-Practice-menu/) |
[Code base](https://github.com/Rybchynskyi/HTML-CSS-Practice-menu)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Rybchynskyi** [comment]
Task "HTML & CSS Practice"

File: submissions/Rybchynskyi/app-Popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
+body {
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/css/style.css:1]: Please, check all this file and format it by one style: add 1 empty line after each block, start each rule from new line and remove necessary comments

File: submissions/Rybchynskyi/app-Popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+}
+
+.link:focus {
+    text-decoration: underline;
+    color: black;
+}
+
+.header__img__menu:last-child {
+
+}
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/css/style.css:58]: remove empty block please

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:8]: Add name of your app

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="header__row">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:13]: Use `nav>ul>li>a` for list of navigation

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+                <a class="link" href="">Mail</a>
+            </div>
+            <div class="header__menu">
+                <a class="link" href="">Images</a>
+            </div>
+            <div class="header__menu for_popup">
+                <label class="check_1">
+                    <img class="header__img__menu" src="img/popup-button.png" alt="menu">
+                    <input class="check_input_1" type="checkbox">
+                    <div class="menu__items">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:24]: Use `ul>li>a` for list of menu

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+                            </div>
+                        </div>
+                        <label class="check_2">
+                            <div class="menu__items__separator">
+                                More
+                            </div>
+                            <input class="check_input_2" type="checkbox">
+                            <div class="menu__items__second__section">
+                                <hr class="menu__items__second__section__hr">
+                                <div class="grid__second__section">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:106]: This is a list too

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+        <div class="header__row">
+            <div class="header__menu">
+                <a class="link" href="">Mail</a>
+            </div>
+            <div class="header__menu">
+                <a class="link" href="">Images</a>
+            </div>
+            <div class="header__menu for_popup">
+                <label class="check_1">
+                    <img class="header__img__menu" src="img/popup-button.png" alt="menu">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:22]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+    <header class="header">
+        <div class="header__row">
+            <div class="header__menu">
+                <a class="link" href="">Mail</a>
+            </div>
+            <div class="header__menu">
+                <a class="link" href="">Images</a>
+            </div>
+            <div class="header__menu for_popup">
+                <label class="check_1">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:21]: Please, add `:focus` state for this element

File: submissions/Rybchynskyi/app-Popup/index.html [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+                            </div>
+                            <div class="grid__item">
+                                <a class="grid__link" href="" tabindex="1">
+                                    <img class="grid__img" src="img/documents.png" alt="documents">
+                                    <p class="grid__name">Document</p>
+                                </a>
+                            </div>
+                        </div>
+                        <label class="check_2">
+                            <div class="menu__items__separator">
```
Comment (HannaSyn) [inline submissions/Rybchynskyi/app-Popup/index.html:100]: Please, add :focus state for this element

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Good work, please, check your navigation with keyboard after you fix comments below to make sure that it is working properly

**Pull request closed without merging**